pyuvdata/uvdata/mir_parser.py

```diff
@@ -967,125 +1754,1691 @@ def from_file(
+            the object. Default is False.
+
+        Raises
+        ------
+        TypeError
+            If attempting to add a MirParser object with any other type of object.
+        ValueError
+            If the objects cannot be combined, either because of differing metadata (if
+            `overwrite=False`), different data being loaded (raw vs vis vs auto), or
+            because the two objects appear to be loaded from different files (and
+            `force=False`).
+        """
+        return self.__add__(
+            other, merge=merge, overwrite=overwrite, force=force, inplace=True
+        )
+
+    def select(
+        self,
+        where=None,
+        and_where_args=True,
+        and_mask=True,
+        update_data=None,
+        reset=False,
+    ):
+        """
+        Select a subset of data inside a Mir-formatted file.
+
+        This routine allows for one to select a subset of data within a Mir dataset,
+        based on various metadata. The select command is designed to be flexible,
+        allowing for both multiple simultaneous selections and serial calls. Users
+        should be aware that the command is case sensitive, and uses "strict" agreement
+        (i.e., it does not mimic the behavior is `np.isclose`) with metadata values.
+        By default, multiple selection criteria are combined together via an "and"
+        operation; for example, if you wanted to select only data from Antenna 1 while
+        on 3c84, you would set `where=(("source", "eq", "3c84"), ("ant", "eq", 1))`.
+
+        The select command will automatically translate information in `codes_data`, the
+        and the various indexing keys, e.g., it will convert an allowed value for
+        "source" (found in`codes_read`) into allowed values for "isource" (found in
+        `in_read`).
+
+        Parameters
+        ----------
+        where : tuple or list of tuples
+            Optional argument, where tuple is used to identify a matching subset of
+            data. Each tuple must be 3 elements in length. The first element should
+            match one of the field names inside one of the metadata attributes (e.g.,
+            "ant1", "mjd", "source", "fsky"). The second element specifies the
+            comparison operator, which is used to compare the metadata field against
+            the third element in the tuple. Allowed comparisons include:
+            "eq" | "==" (equal to, matching any in the third element),
+            "ne" | "!=" (not equal to, not matching any in third element),
+            "lt" | "<" (less than the third element),
+            "le" | "<=" (less than or equal to the third element),
+            "gt" | ">" (greater than the third element),
+            "ge" | ">=" (greater than or equal to the third element),
+            "between" (between the range given by two values in the third element),
+            "outside" (outside of the range give by two values in the third element).
```

Comment:
This is not getting rendered well in RTD. I'd suggest trying adding empty lines before this enumeration, and maybe also adding dashes for bullet points? Will probably need some experimentation.

Reply:
Updated in a2f46b4b -- I _think_ it looks better now, but take a look and tell me what you think.

Reply:
I think it works now. There is a way to make it render as a bulleted list where each type is in its own line. I can try to figure that out if you like, but I think it's ok to leave it as is.